.github/workflows/test-suite-native-e2e-android.yml

```diff
@@ -115,6 +115,14 @@ jobs:
           path: node_modules
           key: node_modules/${{ github.ref }}/${{github.run_id}}
 
+      - name: Install adb (Android SDK Platform Tools)
```

Comment:
Sure, I removed it since `device.reverseTcpPort()` works nicely.

https://github.com/trezor/trezor-suite/pull/14667/commits/d4003508eaa78fb168e359f9eacec13dc32e80e0